backend-rust/src/indexer.rs

```diff
@@ -3654,8 +3732,12 @@ impl PreparedScheduledReleases {
             block_height,
             AccountStatementEntryType::TransferOut,
         )?;
+        let account_address =
+            concordium_rust_sdk::base::contracts_common::AccountAddress::from_str(
+                target_address.as_str(),
+            )?;
```

Comment:
Since we are not storing a string for the account_adress anymore in the various struct, all of current `Arc<String>` types should probably be `CanonicalAccountAddress` and we avoid parsing the string here